tensorflow/compiler/mlir/tensorflow/transforms/einsum.cc

```diff
@@ -73,7 +73,7 @@ bool tokenizeEquation(const llvm::StringRef& equation,
   int index = 0;
   int variable_count = 0;
   llvm::Regex r("[[:alpha:]]");
-  while (index < equation.size()) {
+  while (static_cast<size_t>(index) < equation.size()) {
```

Comment:
I meant changing the type on declaration of local variable (line 73)

Reply:
updated.